localstack-core/localstack/state/codecs.py

```diff
@@ -0,0 +1,63 @@
+"""Factory for encoders and decoders"""
+
+from localstack import config
+from localstack.state import Decoder, Encoder
+from localstack.state.pickle import JsonDecoder, JsonEncoder, PickleDecoder, PickleEncoder
+
+ENCODERS = {
+    "jsonpickle": JsonEncoder,
```

Comment:
q: Should we get rid of the `jsonpickle` encoder? As we decided to move forward with `avro`, it won't be needed anymore. If so, we can remove the decoder (and the dependencies) in a follow-up PR.

Reply:
cool if we don't need it any longer! you're welcome to remove it in a follow up pr :+1: